contentcuration/contentcuration/views/settings.py

```diff
@@ -165,6 +166,7 @@ def form_valid(self, form):
             os.unlink(csv_path)
 
```

Comment:
@bjester sir I have just one question/concern.

On the frontend we require the user to manually delete all channels in which they are the only editor. Should I check the same on the backend here and error out if that's not the case?